aspnet-core/xml/ns-Microsoft.AspNetCore.Antiforgery.Internal.xml

```diff
@@ -1,6 +0,0 @@
-<Namespace Name="Microsoft.AspNetCore.Antiforgery.Internal">
```

Comment:
A bunch of these namespaces don't have any public types in them and currently do not show up in the docs site (going back to 2.1). Do we still need these files?

Reply:
Can these be removed @gewarren?

Reply:
Yes, I think so.